packages/framework/src/Component/Domain/Multidomain/MultiDomainEntityClassProviderInterface.php

```diff
@@ -2,15 +2,28 @@
 
 namespace Shopsys\FrameworkBundle\Component\Domain\Multidomain;
 
+/**
+ * Interface MultiDomainEntityClassProviderInterface
```

Comment:
this line is useless :wink:

Reply:
Okay, PHPstorm generated it for me so i left it there